databricks/koalas/tests/test_series.py

```diff
@@ -1495,6 +1495,17 @@ def test_floordiv_nan(self):
 
         self.assert_eq(repr(pser.floordiv(np.nan)), repr(kser.floordiv(np.nan)))
 
+    def test_mad(self):
+        pser = pd.Series([1, 2, 3, 4], name="Koalas")
+        kser = ks.from_pandas(pser)
+
+        self.assert_eq(pser.mad(), kser.mad())
+
+        pser = pd.Series([None, -2, 5, 10, 50, np.nan, -20], name="Koalas")
+        kser = ks.from_pandas(pser)
+
+        self.assert_eq(pser.mad(), kser.mad())
+
```

Comment:
Can we have a test with `MultiIndex` ?

you can

```python
pmidx = pd.MultiIndex.from_tuples([('x', 'a'), ('x', 'b'), ('y', 'c'), ('y', 'd')])
pser.index = pmidx
kser = ks.from_pandas(pser)

self.assert_eq(pser.mad(), kser.mad())
```

Reply:
Thanks, I added new tests with `MultiIndex`